Formula/kube-aws.rb

```diff
@@ -14,43 +14,61 @@ class KubeAws < Formula
     sha256 "6c5d8d12c01d94918ef68bf4d487aa1b6f577bed3aa64145cd58192fdb8aa948" => :yosemite
   end
 
+  devel do
+    url "https://github.com/coreos/kube-aws/archive/v0.9.3-rc.5.tar.gz"
+    sha256 "0c7d3e6d40adc137d6ff9202400c72b832db4e85ed60abd6dfdb17a9ae301a03"
+    version "0.9.3-rc.5"
+  end
+
   depends_on "go" => :build
 
   def install
     gopath_vendor = buildpath/"_gopath-vendor"
     gopath_kube_aws = buildpath/"_gopath-kube-aws"
-    kube_aws_dir = "#{gopath_kube_aws}/src/github.com/coreos/coreos-kubernetes/multi-node/aws"
+    kube_aws_dir = "#{gopath_kube_aws}/src/github.com/coreos/kube-aws"
 
     mkdir_p gopath_vendor
     mkdir_p File.dirname(kube_aws_dir)
 
-    ln_s buildpath/"multi-node/aws/vendor", "#{gopath_vendor}/src"
-    ln_s buildpath/"multi-node/aws", kube_aws_dir
+    ln_s buildpath/"vendor", "#{gopath_vendor}/src"
+    ln_s buildpath, kube_aws_dir
 
     ENV["GOPATH"] = "#{gopath_vendor}:#{gopath_kube_aws}"
 
     cd kube_aws_dir do
-      system "go", "generate", "./pkg/config"
-      system "go", "build", "-ldflags",
-             "-X github.com/coreos/coreos-kubernetes/multi-node/aws/pkg/cluster.VERSION=#{version}",
-             "-a", "-tags", "netgo", "-installsuffix", "netgo",
-             "-o", bin/"kube-aws", "./cmd/kube-aws"
+      system "go", "generate", "./config"
+      if build.devel? || build.head?
```

Comment:
Flip the `if`/`else` and do `if build.stable?` ... `else`. Don't do `unless` ...`else`. Thanks!